packages/user-operation-controller/package.json

```diff
@@ -41,7 +41,7 @@
     "@metamask/network-controller": "^17.2.0",
     "@metamask/polling-controller": "^5.0.0",
     "@metamask/rpc-errors": "^6.1.0",
-    "@metamask/transaction-controller": "^21.2.0",
```

Comment:
If we bumped this controller we would also want to bump this too:

``` suggestion
    "@metamask/transaction-controller": "^22.0.0",

Reply:
Done in https://github.com/MetaMask/core/pull/3915/commits/78d9aea07deaed854110580a9200c9bcecce7213